packages/app/src/cli/utilities/app/http-reverse-proxy.ts

```diff
@@ -82,10 +83,13 @@ ${output.token.json(JSON.stringify(rules))}
   })
 
   await Promise.all([
-    output.concurrent([...processes, ...additionalProcesses], (abortSignal) => {
-      abortSignal.addEventListener('abort', () => {
-        server.close()
-      })
+    renderConcurrent({
+      processes: [...processes, ...additionalProcesses],
+      onAbort: (abortSignal) => {
```

Comment:
What if we passed the `abortController` instead of the signal? The reason is that we need the controller inside the `renderConcurrent` to be able to call `abort` in the rescue block.